### PR TITLE
Add Ana Castillo Botto to integration

### PR DESCRIPTION
**Pull request by anatron**

### CODE REVIEW

File: modules/users/manifests/anacastillobotto.pp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Creates the anacastillobotto user
+class users::<anacastillobotto> {
+  govuk_user { '<anacastillobotto>':
```
Comment (sivsubra-tw) [inline modules/users/manifests/anacastillobotto.pp:3]: Hi Ana, your class and username needs to be without the angled brackets. Please refer to https://github.com/alphagov/govuk-puppet/pull/12069/commits/1b9a267dc60791f8594ff00d38331a66739f3b84 if it helps!

**Review by sivsubra-tw — CHANGES_REQUESTED** [review]

**Comment by AgaDufrat** [comment]
CI is failing with:
```
Failures:

  1) users::anacastillobotto should have a username of the correct form
     Failure/Error: expect(uname_from_full).to eq(uname_from_email), "name in fullname(#{user[:fullname]}) must match name in email(#{user[:email]})"
       name in fullname(Ana Castillo Botto) must match name in email(ana.botto@digital.cabinet-office.gov.uk)
     # ./modules/users/spec/classes/users_spec.rb:39:in `block (3 levels) in <top (required)>'
```

File: hieradata_aws/integration.yaml [hunk 2]
```diff
@@ -283,6 +283,7 @@ users::usernames:
   - alessiatosi
   - alexnewton
   - alistairdavidson
+  - anacastillobotto
```
[the remark this answers is not in the document]
Comment (anatron) [reply hieradata_aws/integration.yaml:286]: @AgaDufrat Hi Aga! Thank you for letting me know! I will correct it now :) The struggle is real

File: modules/users/manifests/anabotto.pp [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Creates the anabotto user
+class users::anabotto {
+  govuk_user { 'anabotto':
+    fullname => 'Ana Castillo Botto',
```
Comment (robinjam) [inline modules/users/manifests/anabotto.pp:4]:
```suggestion
    fullname => 'Ana Botto',
```

Unfortunately our CI requires your full name to match your email address (though I'm not sure why...)

**Review by robinjam — CHANGES_REQUESTED** [review]

**Review by robinjam — APPROVED** [review]
Tests are passing now, so LGTM!

**Pull request merged**